package.json

```diff
@@ -29,5 +29,27 @@
   },
   "lint-staged": {
     "**/*": "prettier --workspaces --if-present --write --ignore-unknown"
+  },
+  "overrides": {
+    "@injectivelabs/sdk-ts@1.10.72": {
+      "@injectivelabs/token-metadata": "1.10.42"
+    },
+    "@ledgerhq/hw-transport@6.27.1": {
+      "@ledgerhq/errors": "6.10.2",
+      "@ledgerhq/logs": "6.12.0",
+      "@ledgerhq/devices": "6.27.1"
+    },
+    "@ledgerhq/hw-transport-webhhid@6.27.1": {
+      "@ledgerhq/errors": "6.10.2",
+      "@ledgerhq/logs": "6.12.0",
+      "@ledgerhq/devices": "6.27.1"
+    },
+    "@wormhole-foundation/sdk-definitions-ntt": {
```

Comment:
Should probably have QA do a few NTT transfers just to make sure that nothing broke (I wouldn't think so, but I guess to make sure)